src/Couchbase.Lite.Shared/Support/iOS/iOSReachability.cs

```diff
@@ -96,9 +96,9 @@ private void NotifyFlagsChanged(NetworkReachabilityFlags flags)
             });
         }
 
-        #endregion
```

Comment:
VS is automatically change the indentation when I just modify the code. I will change them back before merge this PR.

Reply:
This is a setting you can modify